src/Breadcrumbs.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import styled from 'styled-components'
+import {COMMON, FLEX, get} from './constants'
+import theme from './theme'
+
+const ITEM_CLASS = 'Breadcrumb-item'
+const SELECTED_CLASS = 'selected'
+
+function BreadcrumbBase({className, children, label, separator, ...rest}) {
+  const classes = classnames(className, 'Breadcrumb')
+  const wrappedChildren = React.Children.map(children, (child) => <Breadcrumb.Wrapper separator={separator}>{child}</Breadcrumb.Wrapper>)
+  return (
+    <nav className={classes} aria-label={label} {...rest}>
+      <ol>{wrappedChildren}</ol>
+    </nav>
+  )
+}
+
+const Breadcrumb = styled(BreadcrumbBase)`
+  display: flex;
+  justify-content: space-between;
+  ${COMMON};
+  ${FLEX};
+  & > ol {
+    padding-left: 0;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+`
+
+Breadcrumb.Wrapper= styled.li.attrs(({separator}) => ({
+  className: classnames(ITEM_CLASS),
+  separator
```

Comment:
Let's remove the `separator` prop in favor of standardizing `/` being used in breadcrumbs. Since our goal is consistent UI I think we should keep this part of the UI pattern less flexible.

Reply:
Additionally, we'll want to add an `aria-hidden` attribute to the content so that screen readers know it's decorative content (this isn't currently done in Primer CSS but it should be! :) ) I'm happy to make this change if you'd like! :)

Reply:
sorry I am not the most fluent with how to properly add the aria properties... should this be on the `nav` or on the content?

Reply:
It should be on the content! I can make that change :)